src/lib/Hydra/Schema/ResultSet/TaskRetries.pm

```diff
@@ -0,0 +1,71 @@
+package Hydra::Schema::ResultSet::TaskRetries;
+
+use strict;
+use warnings;
+use utf8;
+use base 'DBIx::Class::ResultSet';
+use List::Util qw(max);
+use Hydra::Math qw(exponential_backoff);
+use Hydra::Task;
+
+=head2 get_seconds_to_next_retry
+
+Query the database to identify how soon the next retryable task is due
+for being attempted again.
+
+If there are no tasks to be reattempted it returns undef.
+
+If a task's scheduled retry has passed, it returns 0.
+
+Otherwise, returns the number of seconds from now to look for work.
```

Comment:
This just reads kinda weird to me...

Maybe, `returns the number of seconds to wait before looking for more work`? (Is that even an accurate description?)

Reply:
```suggestion
Otherwise, returns the number of seconds to wait before looking for more work.
```